hardhat-scripts/helpers/createERC20ContractInstance.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable */
+
+const ethers = require("ethers");
+const VotingTokenArtifact = require("@uma/core/build/contracts/VotingToken.json");
```

Comment:
nit: we should drop uma/core if possible.

Can we use
```ts
// Use contracts-node because this script is being run in node.
const { getAbi, getAddress } = require("@uma/contracts-node");

const abi = getAbi("ERC20");
const address = await getAddress("VotingToken", networkId);
```

Reply:
Totally fair. This was from my previous iteration on hardhat-test which hasn't updated yet.